data/human/human missions.txt

```diff
@@ -3601,3 +3601,80 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+
+
+
+mission "Repulsor failure"
+	landing
+	# Don't require the player to pay credits again if the player was already allowed to land when this mission triggered.
+	clearance
+	to offer
+		random < 8
+		credits >= 2000000
+	on offer
+		conversation
+			"You see a warning that one of the repulsors on your ship is not working when you start de-orbiting."
+			choice
+				"(Return to orbit to fix the issue in space.)"
+					launch
+				"(Attempt to land anyway.)"
+			"Orbital entry is a bit more steep than normal and your ship is starting to lean a bit."
+			choice
+				"(Try to return to orbit to still fix the issue in space.)"
+					goto reorbit
+				"(Continue the attempt to land.)"
+			"You're reaching the point where your ship will not be able to re-orbit on its own power."
+			"The cockpit is filled with warnings about the descent angle and attitude control."
+			choice
+				"(Attempt to re-orbit and hope that the ship still makes it.)"
+					goto reorbit
+				"(Proceed with landing, crashing, or dying. Whichever one it's gonna be.)"
+			"Your decent is still far more steep than normal, and the ship is starting to roll."
+			"It is no longer possible to correct both your descend angle and maintain attitude control."
+			choice
+				"(Prioritize roll stabilization.)"
+					goto rolling
+				"(Prioritize lowering the descent angle.)"
+				"(Switch to full auto-pilot control.)"
+			"Your ship begins to roll faster and faster, but the autopilot somehow manages to soften the descent angle."
+			choice
+				"(Prioritize roll stabilization now.)"
+					goto rolling
+				"(Continue prioritizing lowering the descent angle.)"
+				"(Let the autopilot handle the landing.)"
+			branch miracle
+				random < 80
+			"Unfortunately it is not enough, the ship starts rolling even faster. You lose consciousness before your ship crashes and you die."
```

Comment:
Seems a bit clumsy for something that - even with a touch of comedy - is supposed to have some form of impact. I've seen other writers use kind of an "open ended" sentence, similar to "your vision blurs, before consciousness slip from you..."

Reply:
Agreed, I updated the text.